### DISCUSSION TITLE
ATtiny88 Micronucleus Upload Error

### DISCUSSION THREAD

**Discussion by founfabug** [SpenceKonde/ATTinyCore#710] [Support / Q & A]
Arduino: 1.8.13
ATTinyCore-2.0.0-dev: From today
System: Windows 10
Error on uploading:
`java.io.IOException: Cannot run program "C:\Users\...\AppData\Local\Arduino15\packages\ATTinyCore\tools\micronucleus\2.5-azd1b/launcher": CreateProcess error=2, The system cannot find the file specified`

`micronucleus.exe` is at that location.

Note that `platform.txt` contains a definition: 
`tools.micronucleus.cmd.path={runtime.tools.micronucleus.path}/launcher` 
which differs from master branch: 
`tools.micronucleus.cmd.path={runtime.tools.micronucleus.path}/micronucleus `

Reverting the definition in `platform.txt` resolves the error, but introduces `Unrecognized option: -cdigispark`

Additional debugging indicates other things are not quite right, including my understanding of what is going on :-)

Uploads as expected with 1.5.2 core.

_Thank you_

**Comment by SpenceKonde** [comment]
I agree, this looks bad. 

Please continue discussion in #711. 

Edit: locking to keep all the discussion of what is a cler bug in the issue, instead of discussions.
